From: Pino Toscano <pino@kde.org>
Date: Sun, 14 Feb 2016 07:49:09 +0000
Subject: Fix flags for KFontDialog::getFont()
X-Git-Url: http://quickgit.kde.org/?p=krecipes.git&a=commitdiff&h=f6d4f709ec57835b3fa4a660239a07321c9d02ff
---
Fix flags for KFontDialog::getFont()

Pass a proper value for flags, instead of a boolean.

This fixes the build with GCC 6.
---


--- a/src/dialogs/setupdisplay.cpp
+++ b/src/dialogs/setupdisplay.cpp
@@ -562,7 +562,7 @@
 void SetupDisplay::setFont()
 {
 	KreDisplayItem *item = *node_item_map->find( m_currNodeId );
-	if ( KFontDialog::getFont( item->font, false, view() ) == QDialog::Accepted ) {
+	if ( KFontDialog::getFont( item->font, KFontChooser::NoDisplayFlags, view() ) == QDialog::Accepted ) {
 		m_currentItem = item;
 		loadFont(m_currNodeId,item->font);
 		m_currentItem = 0;

